IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -45,6 +45,7 @@
 #import "MSIDAppMetadataCacheQuery.h"
 #import "MSIDGeneralCacheItemType.h"
 #import "MSIDIntuneEnrollmentIdsCache.h"
+#import "MSIDRequestContext.h"
```

Comment:
Isn't this already imported in pch file?

Reply:
we don't use your pch, that's why I needed to add a bunch of these

Reply:
No, it's included in `MSIDLogger+Internal.h`. We don't include it, because we overwrote it

Reply:
Should we add it to pch file then? Basically, we need a future proof way of importing this, otherwise iOS team will continue forgetting this.